Dockerfile

```diff
@@ -41,15 +41,14 @@ RUN rm -rf /var/lib/apt/lists/* && \
         | tar -xz -C /opt/ants-2.3.4 --strip-components 1
 
 RUN pip install -U pip
```

Comment:
Is this necessary?

Reply:
Nope, you can remove

Reply:
Done in [6888d13](https://github.com/scil-vital/dockerfile_tractolearn/pull/7/commits/6888d13dd817844f74983c5d0115dd5075b4e1f2).